python/pyproject.toml

```diff
@@ -19,6 +19,7 @@ dependencies = [
     "requests >= 2.31.0",
     "gql[all] == 3.5.0",
     "matplotlib >= 3.4.3",
+    "ipywidgets >= 8.1.5",
```

Comment:
is this accidental? If it isn't just ignore this

Reply:
It is another fix that I shoved in here :)